### PR TITLE
[v0.24.4] devtool: add check_ssbd() to checkenv()

### PR DESCRIPTION
**Pull request by alindima**
Signed-off-by: alindima <alindima@amazon.com>

# Reason for This PR

`[Author TODO: add issue # or explain reasoning.]`

## Description of Changes

`[Author TODO: add description of changes.]`

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [ ] All commits in this PR are signed (`git commit -s`).
- [ ] The reason for this PR is clearly provided (issue no. or explanation).
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

### CODE REVIEW

File: tools/devtool [hunk 1]
```diff
@@ -969,6 +969,23 @@ check_swap () {
...
+        local param="spec_store_bypass_disable=on"
+    fi
+
+    ssbd_sysfs_file="/sys/devices/system/cpu/vulnerabilities/spec_store_bypass"
+
+
+    (test -f $ssbd_sysfs_file) && (grep -q "^Vulnerable" $ssbd_sysfs_file) && \
+      say_warn "WARNING: SSBD mitigation is either globally disabled or"\
+      "system does not support mitigation via prctl or seccomp. Try enabling"\
+      "it system-wide, using the \`${param}\` boot parameter."
```
Comment (acatangiu) [inline tools/devtool:986]: This doesn't warn if `ssbd_sysfs_file` doesn't exist

**Comment by acatangiu** [comment]
process nit: usually we fix smth in `main` then backport to other branches to have a consistent commit propagation model.

File: tools/devtool (hunk 1 above)
Comment (alindima) [reply tools/devtool:986]: If this file is not present, the kernel does not support this mitigation. I think this shouldn't happen on kernels newer than 4.14. Is this correct @georgepisaltu ?

Comment (alindima) [reply tools/devtool:986]: I'll update this nevertheless, to be on the safe side

Comment (acatangiu) [reply tools/devtool]: yes, pls cover that case even if it's not supported, it only costs us one line here 😄

**Review by acatangiu — DISMISSED** [review]
Since we'll do the release at this commit, I think we should also rename the commit to include `release 0.24.4` or smth, otherwise git log history will be confusing.

**Comment by alindima** [comment]
> Since we'll do the release at this commit, I think we should also rename the commit to include `release 0.24.4` or smth, otherwise git log history will be confusing.

done, PTAL

**Review by acatangiu — APPROVED** [review]

**Review by gc-plp — APPROVED** [review]

**Pull request merged**